config/initializers/session_store.rb

```diff
@@ -21,7 +21,7 @@
                 when "memory" then  :memory_store
                 when "cache" then   :dalli_store
                 else
-                  raise "session_store, '#{evm_store}', invalid. Should be one of 'sql', 'memory', 'cache'"
+                  raise "session_store, '#{evm_store}', invalid. Should be one of 'sql', 'memory', 'cache'.  Source configuration: #{config_file}"
```

Comment:
Specifically, I saw this

```
---
log_depot:
  uri: smb://server/share
  username: user
  password: v2:{...}
```

And git grep'd "server/share" and found the problem right away.